src/csharp/Microsoft.Spark/Sql/Functions.cs

```diff
@@ -3335,6 +3502,35 @@ public static Column Window(Column column, string windowDuration)
             return ApplyFunction("window", column, windowDuration);
         }
 
+        /// <summary>
+        /// Generates session window given a timestamp specifying column.
+        /// </summary>
+        /// <param name="timeColumn">The column or the expression to use as the timestamp for windowing by 
+        /// time</param>
+        /// <param name="gapDuration">A string specifying the timeout of the session, e.g. '10 minutes',
+        /// '1 second'</param>
+        /// <returns>Column object</returns>
+        [Since(Versions.V3_2_0)]
+        public static Column SessionWindow(Column timeColumn, string gapDuration)
```

Comment:
Shouldn't these `SessionWindow` methods be named `Session_Window` ?  You followed that convention for `Bitwise_Not` and `Count_Distinct`.

Although it does seem like we didn't follow this convention initially when first porting these APIs.   ie, `next_day` (scala) => `NextDay` (c#).  I think we should probably start following it moving forward to avoid any naming issues in the future.

Reply:
Yeah I did that for `Count_Distinct` and `Bitwise_NOT` coz there were other functions with the name `CountDistinct` and `BitwiseNOT`. But yes I agree we should follow this going forward so changed to `Session_Window`